tests/test_wallet/test_wallet_settings.py

```diff
@@ -3,6 +3,7 @@
 from allure import step
 
 import configs
+from constants.wallet import WalletNetworkSettings
```

Comment:
I would move this import below 'import driver', usually we group imports this way (for example like here - https://github.com/status-im/desktop-qa-automation/blob/master/tests/test_communities.py)